.github/workflows/main.yml

```diff
@@ -408,7 +408,7 @@ jobs:
   # github releases and/or tags for pushes.
   publish:
     name: Publish
-    needs: [doc_book, doc_api, wheels, build]
+    needs: [doc_book, doc_api, wheels, build, wheels]
```

Comment:
I think `wheels` is already listed here?

Reply:
Oeps! Fixed